File: app/src/main/java/com/gmail/fattazzo/formula1world/domain/F1Circuit.java
package com.gmail.fattazzo.formula1world.domain;

import java.io.Serializable;

/**
 * @author fattazzo
 *         <p/>
 *         date: 06/06/17
 */
public class F1Circuit implements Serializable {

    public String circuitRef;

    public String url;

    public String name;

    public F1Location location;

    @Override
    public String toString() {
        return "F1Circuit{" +
                "circuitRef='" + circuitRef + '\'' +
                ", url='" + url + '\'' +
                ", name='" + name + '\'' +
                ", location=" + location +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/domain/F1Circuit.kt
package com.gmail.fattazzo.formula1world.domain

import java.io.Serializable

/**
 * @author fattazzo
 *
 *
 * date: 06/06/17
 */
class F1Circuit : Serializable {

    var circuitRef: String? = null

    var url: String? = null

    var name: String? = null

    var location: F1Location? = null

    override fun toString(): String {
        return "F1Circuit{" +
                "circuitRef='" + circuitRef + '\'' +
                ", url='" + url + '\'' +
                ", name='" + name + '\'' +
                ", location=" + location +
                '}'
    }
}
